about/process/ppa.rst

```diff
@@ -1,8 +1,16 @@
 repo.ubports.com
 ================
 
-This is the package archive system for UBports projects. It hosts various PPAs containing all deb-components of Ubuntu Touch. New PPAs can be created dynamically by the CI server using a special :doc:`git-branch naming convention <branch-naming>`.
+This is the package archive system for UBports projects. It hosts various PPAs containing all deb-components of Ubuntu Touch.
 
-The name of the branch translates literally to the name of the PPA: ``http://repo.ubports.com/dists/[branch name]``
+Package-naming convention
+-------------------------
+
+The name of the deb-package and the name of the git-repository should be the same.
```

Comment:
This isn't exactly true. Our repositories are named by the name of the *source* package. In some cases, they have the `-packaging` moniker to denote that the actual upstream source is retrieved from somewhere else.